Assets/Scripts/Quest/NextStageButton.cs

```diff
@@ -32,22 +41,22 @@ public void Update()
         DialogBox db;
         if (Game.Get().quest.phase == Quest.MoMPhase.horror)
         {
-            db = new DialogBox(new Vector2(UIScaler.GetHCenter(-6f), UIScaler.GetBottom(-2.5f)), new Vector2(16, 2), "Horror Phase");
+            db = new DialogBox(new Vector2(UIScaler.GetHCenter(-6f), UIScaler.GetBottom(-2.5f)), new Vector2(16, 2), HORROR_STEP);
```

Comment:
The horror phase translation text is inherited from ffg phases text. It uses ALL CAPS for the 2 main phases in the game. Players TURN and Mithos TURN. And in Camel the 2 subphases of the Mithos turn: Horror Step and Monster Step.
I decided it in order to have the text in all languages, but I can remove other langs and Camel the texts